core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/DistinctAggregationStrategyChooser.java

```diff
@@ -17,9 +17,17 @@
 import io.trino.cost.PlanNodeStatsEstimate;
```

Comment:
Change the commit message DistinctAggregationController  -> DistinctAggregationStrategyChooser